src/components/Send/SendScreen.js

```diff
@@ -750,12 +754,14 @@ class SendScreen extends Component<Props, State> {
             assetsMetadata={availableAssets}
             unselectEnabled={false}
           />
+          {this.state.fee == null &&
```

Comment:
If the fee is already set, it doesn't show the indicator (from what I see). But if we change the amount it could be changing the number of utxos used so it would require to re-compute and we need to show the indicator

Reply:
any time the amount change it recomputes the fee and shows the indicator. The utxos only change when you receive a new utxo, they are not the utxos used for building the tx.

Reply:
@v-almonacid 

a) I didn't see where this nullifies the fee if the amount to send is different
b) About the utxos, when you are going to send X amount you pick UTXOs and you not necessarily use all of your utxos, if you change the amount it may require more UTXOs and that would make the tx bigger in size therefore more expensive

Reply:
a) is reset in `revalidate()`

b) Yes. That's the behaviour. Am I missing something?

Reply:
I can verify that it recomputes the fees as well and nullifies the fees after changing the amount

Reply:
@v-almonacid 
a) okay, great
b) I wanted to clarify that statement `The utxos only change when you receive a new utxo`, but if a) is true it should be covered!

Reply:
ah ok! I thought you were referring to `this.props.utxos` which is the full utxo set